### PR TITLE
feat: enable mysql logging on slow queries

### PR DESCRIPTION
**Pull request by aaronreynoza**

### CODE REVIEW

**Review by dfry — APPROVED** [review]
lgtm

File: terraform/k8s/default-config/mojaloop-stateful-resources.json [hunk 1]
```diff
@@ -340,7 +348,19 @@
...
                 "is_legacy": false,
                 "existing_secret": "mysql-central-ledger-secret",
+                "general_log": 1,
+                "slow_query_log": 1,
                 "root_password": "",
+                "default_authentication_plugin": "mysql_native_password",
+                "basedir": "/opt/bitnami/mysql",
+                "plugin_dir": "/opt/bitnami/mysql/lib/plugin",
+                "socket": "/opt/bitnami/mysql/tmp/mysql.sock",
+                "datadir": "/bitnami/mysql/data",
```
Comment (dfry) [inline terraform/k8s/default-config/mojaloop-stateful-resources.json:358]: do all of these params need to be exposed as params?

Comment (aaronreynoza) [reply terraform/k8s/default-config/mojaloop-stateful-resources.json:358]: @dfry, we need to be able to change the values if needed. Default values will continue to be the same, but users will be able to change them from implementations now.

**Review by dfry — APPROVED** [review]
approved

**Pull request merged**